blocks/v2-video/v2-video.css

```diff
@@ -0,0 +1,117 @@
+/* stylelint-disable selector-class-pattern */
+.v2-video-wrapper {
+  --video-inset: inset(32px 48px round 8px);
+
+  padding: 0 !important;
+  color: var(--c-white);
+  margin: 0 !important;
+  width: 100% !important;
+}
+
+.v2-video {
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  clip-path: inset(24px 24px round 8px);
+  transition: clip-path var(--duration-large) ease-in-out;
```

Comment:
We’re missing a delay, and the animation itself should take longer than 320ms.

As we can’t edit in Figma, I can’t see the timings/easing being used. So we need to align with the agency on the right timing and easing.

Reply:
I leave it as it is now. When we get the timing I'll update it.